src/System.Data.Common/src/System/Data/Common/DbConnection.cs

```diff
@@ -35,6 +35,8 @@ protected DbConnection() : base()
         /// </summary>
         protected virtual DbProviderFactory DbProviderFactory => null;
 
+        internal DbProviderFactory ProviderFactory=>DbProviderFactory;
```

Comment:
Will fix the space issue. 

The internal member is important as DbProviderFactories (the class I'll try to bring back next) is using this member on .NET full to obtain the factory in its method GetFactory(DbConnection), so to provide the same interface on .net core, this method is needed. It's currently indeed 'dead' as nothing references it. A comment is indeed needed here.